config/crd/bases/rbac-operator.argoproj-labs.io_argocdroles.yaml

```diff
@@ -40,13 +40,16 @@ spec:
             description: ArgoCDRoleSpec defines the desired state of Role
             properties:
               rules:
+                description: Rules define the desired set of permissions.
                 items:
                   properties:
                     objects:
+                      description: List of target objects.
                       items:
                         type: string
                       type: array
                     resource:
+                      description: Target resource type. 
```

Comment:
@jameskim0987 could you please add those comments (objects, resource, verbs, ...) to the api/v1alpha1/argocdrole_types.go? otherwise the changes will be overriden, if we change something in the types file and generate the yaml

Reply:
Sure thing! Done.